docs/source/contributing/developing-core.md

```diff
@@ -104,15 +104,16 @@ For the fish shell, see [nvm.fish](https://github.com/jorgebucaran/nvm.fish).
 1.  Install or update the supported LTS versions of Node.js, then activate the version supported in Volto.
 
     ```shell
-    nvm install "lts/*"
-    nvm use 20
+    nvm install 21.5.0
+    nvm use 21.5.0
     ```
 
 2.  Verify that the supported version of Node.js is activated.
 
     ```shell
     node -v
     ```
+    should see 21.5.0(latest)
```

Comment:
Remove.

We need @sneridagh to comment on whether this step 2 is even correct, per https://github.com/plone/volto/issues/5556#issuecomment-1864801394.

```suggestion
```

Reply:
@stevepiercy ,on  my mac ,running `node --v` shows  `bad practise` so should I update it to `node --version`?

Reply:
No. Try `node -v`.